### PR TITLE
Update pub/sub diagram in docs

### PR DESCRIPTION
**Pull request by mavam**
This PR makes it easier to understand the difference visually between Cribl's routes and Tenzir's pub/sub approach.

### CODE REVIEW

File: contrib/tenzir-plugins
Comment (dominiklohmann) [inline contrib/tenzir-plugins:1]: This was likely committed by accident.

**Review by dominiklohmann — APPROVED** [review]

File: contrib/tenzir-plugins
Comment (mavam) [reply contrib/tenzir-plugins:1]: Indeed.

Comment (dominiklohmann) [reply contrib/tenzir-plugins:1]: In your rebase that reverted this you also removed the new SVG.

Comment (mavam) [reply contrib/tenzir-plugins:1]: 🙈

**Pull request merged**